build.psm1

```diff
@@ -1206,8 +1216,14 @@ function Start-PSBootstrap {
 function Start-PSPackage {
     [CmdletBinding()]param(
         # PowerShell packages use Semantic Versioning http://semver.org/
+        [Parameter(ParameterSetName = "Version")]
         [string]$Version,
 
+        [Parameter(ParameterSetName = "ReleaseTag")]
+        [ValidatePattern("^v\d+\.\d+\.\d+-\w+\.\d+$")]
```

Comment:
The `-\w+\.\d+` part should be optional - an official release won't have that part I think.